erpnext/stock/doctype/stock_entry/stock_entry.py

```diff
@@ -1072,7 +1072,7 @@ def get_items(self):
 		self.set_actual_qty()
 		self.update_items_for_process_loss()
 		self.validate_customer_provided_item()
-		self.calculate_rate_and_amount()
+		self.calculate_rate_and_amount(raise_error_if_no_rate=False)
```

Comment:
I don't think this is a good solution to the problem you described. This will just delay the error till the last moment of submission, even when doing normal stock entries.  

When is the valuation rate not available? Mostly it's when the item is newly added but doesn't have any stock. In which case the error will be thrown because this is negative stock, this makes total sense as you should be manufacturing using RM which isn't in stock. Unable to save draft stock entry isn't a big deal here IMO. 


Any other use case where this is breaking?